src/main/java/com/github/wovnio/wovnjava/WovnServletFilter.java

```diff
@@ -32,8 +31,9 @@ public void doFilter(ServletRequest request, ServletResponse response, FilterCha
         Headers headers = new Headers((HttpServletRequest)request, settings);
         String lang = headers.getPathLang();
         boolean hasShorterPath = lang.length() > 0 && lang.equals(settings.defaultLang);
-
-        if (!isHtml(headers.pathName)) {
+        boolean canNotTranslate = !htmlChecker.canTranslatePath(headers.pathName)
```

Comment:
I would prefer `canTranslate = htmlChecker.canTranslatePath(headers.pathName) && htmlChecker.canTranslateContentType(response.getContentType())` and use `if (!canTranslate)` in the condition check